docs/source/en/using-diffusers/diffedit.md

```diff
@@ -193,33 +196,38 @@ Finally, pass the embeddings to the [`~StableDiffusionDiffEditPipeline.generate_
 
 ```diff
   from diffusers import DDIMInverseScheduler, DDIMScheduler
-  from diffusers.utils import load_image
+  from diffusers.utils import load_image, make_image_grid
+  from PIL import Image
 
   pipeline.scheduler = DDIMScheduler.from_config(pipeline.scheduler.config)
   pipeline.inverse_scheduler = DDIMInverseScheduler.from_config(pipeline.scheduler.config)
 
   img_url = "https://github.com/Xiang-cd/DiffEdit-stable-diffusion/raw/main/assets/origin.png"
-  raw_image = load_image(img_url).convert("RGB").resize((768, 768))
-
+  raw_image = load_image(img_url).resize((768, 768))
 
   mask_image = pipeline.generate_mask(
       image=raw_image,
+-     source_prompt=source_prompt,
+-     target_prompt=target_prompt,
 +     source_prompt_embeds=source_embeds,
 +     target_prompt_embeds=target_embeds,
   )
 
   inv_latents = pipeline.invert(
+-     prompt=source_prompt,
 +     prompt_embeds=source_embeds,
       image=raw_image,
   ).latents
 
-  images = pipeline(
+  image = pipeline(
       mask_image=mask_image,
       image_latents=inv_latents,
+-     prompt=target_prompt,
+-     negative_prompt=source_prompt,
 +     prompt_embeds=target_embeds,
 +     negative_prompt_embeds=source_embeds,
-  ).images
-  images[0].save("edited_image.png")
+  ).images[0]
+  make_image_grid([raw_image.resize((512, 512)), Image.fromarray((mask_image.squeeze()*255).astype("uint8"), "L").resize((512, 512)), image.resize((512, 512))], rows=1, cols=3)
```

Comment:
Same comment about this being a bit difficult to read

Reply:
Done.